openmmtools/multistate/multistatesampler.py

```diff
@@ -1759,7 +1759,8 @@ def _update_timing(self, iteration_time, partial_total_time, run_initial_iterati
         # TODO: use units for timing information to easily convert between seconds and days
         # there are some mcmc_moves that have no timestep attribute, catch exception
         try:
-            current_simulated_time = self.mcmc_moves[0].timestep * self._iteration * self.mcmc_moves[0].n_steps
+            current_simulated_time = self.mcmc_moves[0].timestep * self._iteration * self.mcmc_moves[0].n_steps * \
```

Comment:
Shouldn't we actually be summing over _all_ `mcmc_moves`, in case there are multiple `MCMCMove` objects with dynamics involved, or appearing in different order?

And something still seems off here---do we want to use the estimate from `average_seconds_per_iteration` instead of the `partial_total_time` (which is the accumulated time since `run()` was started)?